apps/web/src/services/dashboards/widgets/cost-widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue

```diff
@@ -122,6 +122,7 @@ const displayState = reactive({
 });
 
 const [recentSpentPeriod] = useDateRangeFormatter({
+    start: computed(() => widgetState.dateRange.end),
```

Comment:
nitpick: !!!

```typescript
start: computed(() => widgetState.dateRange.start),
```

Reply:
It is intended! This is related to `useDateRangeFormatter` spec!